src/screens/SubmitRatingScreen.tsx

```diff
@@ -36,34 +29,31 @@ const SubmitRatingScreen: React.FC<SubmitRatingScreenProps> = ({
 				}}
 			/>
 			{PLACE_ATTRIBUTES.map((attribute, index) => {
-				const interimRating = interimRatingRef.current[index];
+				const { counter, increment, decrement } = useCounter(DEFAULT_INTERIM_RATING);
```

Comment:
unfortunately, this breaks one of the react [rules of hooks](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level). we shouldn't call hooks in loops, only at the top level. maybe we would be better off just hardcoding 4 of these hooks

Reply:
Totally forgot about this rule, thank you for pointing it out!